src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -684,7 +686,7 @@
 		</type>
 
 		<!-- mini-generic-sharing.c -->
-		<type fullname="Mono.ValueTuple" preserve="fields"/>
+		<!-- Mono.ValueTuple has no fields  <type fullname="Mono.ValueTuple" preserve="fields"/>-->
 		<type fullname="Mono.ValueTuple`1" preserve="fields"/>
```

Comment:
'fields' is used here to preserve the type.

Reply:
and for consistency.

Reply:
The reason I'm trying to change this one is that linker generates a warning if the type has no fields but the descriptor says `preserve="fields"`.

We could change the linker, or we could change the descriptor to `<type fullname="Mono.ValueTuple" required="true"/>`.

I don't feel strongly either way...

Reply:
I changed this to `required="true"`